src/main/java/org/maxgamer/rs/core/Core.java

```diff
@@ -110,87 +117,16 @@ public void run() {
-                if (main.lastModified() != cacheCfg.getLong("modified." + main.getName()) || xtea.lastModified() != cacheCfg.getLong("modified." + xtea.getName())) {
-                    Log.debug("Cache change detected. Recalculating!");
-                    //So we must scan through all of the map files, attempt to parse them, and blacklist broken ones
-                    for (int x = 0; x < 256; x++) {
-                        for (int y = 0; y < 256; y++) {
-                            Reference ref;
-                            try {
-                                ref = r.getReferenceByHash("l" + x + "_" + y);
-                            } catch (FileNotFoundException e) {
-                                continue;
-                            }
-                            try {
-                                MapCache.getObjects(x, y);
-                            } catch (IOException e) {
-                                //File is broken or encrypted and we don't have the key.
-                                r.remove(ref.getId());
-                                //Blacklist the file
-                                cacheCfg.set("encryptedMaps." + ref.getId(), ref.getId());
-                            }
-                        }
-                    }
-                    // Update config & save
-                    cacheCfg.set("modified." + main.getName(), main.lastModified());
-                    cacheCfg.set("modified." + xtea.getName(), xtea.lastModified());
-                    cacheCfg.save();
-                } else {
-                    // We previously worked on this cache, and listed all broken files.
-                    // This is faster than testing each file if it's broken or not.
-                    for (String refId : cacheCfg.getSection("encryptedMaps").getKeys()) {
-                        int referenceId = cacheCfg.getInt("encryptedMaps." + refId, -1);
-                        r.remove(referenceId);
-                    }
-                }
-
-                // Now we re-encode the raw version of the data
-                r.setVersion(r.getVersion() + 1);
-                f.setData(r.encode()); //Set the file data to the reference table
-                cache.setRaw(255, IDX.LANDSCAPES, f.encode());
-                cache.rebuildChecksum();
-            } catch (IOException e) {
-                LOGGER.error("There was an error loading the cache. Please ensure you placed it in the cache/ folder and that it is in tact and readable.", e);
-                System.exit(2);
-            }
-        }
-
+    public static Cache getCache() {
         return cache;
     }
 
-
     /**
      * Fetches the console command sender
      *
      * @return The console as a command sender.
      */
     public static ConsoleSender getConsole() {
-        if (console == null) {
-            console = new ConsoleSender();
-        }
-
         return console;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "modified." 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://titan.maxgamer.org:9000/coding_rules#rule_key=squid%3AS1192)